<logs><log LogGUID="c5a2e92b-b801-4f3a-a630-b50fa3b19e33" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 3:58:55 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>8ec5469b-d182-4e41-bb72-e708198f64b6</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>6wv5vouV4XwTTOxOPPEXzg==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Controllers.HostController.GetSettingsFromDatabase()</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="4f708344-d452-4caa-aa69-47692717fd0b" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 3:58:55 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>77172c4a-d7fe-4cbc-a5e9-8572359b8cee</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>sEjH5O14TFwEfLquEOPxMg==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Portals.PortalAliasController.&lt;GetPortalAliasesInternal&gt;b__11(CacheItemArgs c)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromDictionary(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="919f765d-e9d8-4ea4-a706-622b06b046ec" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 3:58:55 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>Url Processing Exception</PropertyName><PropertyValue>Exception in Url Rewriting Process</PropertyValue></LogProperty><LogProperty><PropertyName>Http Status</PropertyName><PropertyValue>500 Internal Server Error</PropertyValue></LogProperty><LogProperty><PropertyName>Original Path</PropertyName><PropertyValue>null</PropertyValue></LogProperty><LogProperty><PropertyName>Raw Url</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>Final Url</PropertyName><PropertyValue>null</PropertyValue></LogProperty><LogProperty><PropertyName>Rewrite Result</PropertyName><PropertyValue>[no rewrite]</PropertyValue></LogProperty><LogProperty><PropertyName>Redirect Location</PropertyName><PropertyValue>[no redirect]</PropertyValue></LogProperty><LogProperty><PropertyName>Action</PropertyName><PropertyValue>Continue</PropertyValue></LogProperty><LogProperty><PropertyName>Reason</PropertyName><PropertyValue>Exception</PropertyValue></LogProperty><LogProperty><PropertyName>Portal Id</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>Tab Id</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>Http Alias</PropertyName><PropertyValue>Null</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 1</PropertyName><PropertyValue>Exception: Value cannot be null.
Parameter name: source</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 2</PropertyName><PropertyValue>(TRUNCATED TO 500 CHARS): Stack Trace:    at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasLookupInternal(String alias)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasInternal(String httpAlias)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequestForWebServer(String requestedPath)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequest(UrlAction result, String requ</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 3</PropertyName><PropertyValue>Inner Ex : null</PropertyValue></LogProperty><LogProperty><PropertyName>Exception Type</PropertyName><PropertyValue>System.ArgumentNullException</PropertyValue></LogProperty><LogProperty><PropertyName>Message</PropertyName><PropertyValue>Value cannot be null.
Parameter name: source</PropertyValue></LogProperty><LogProperty><PropertyName>Stack Trace</PropertyName><PropertyValue>(TRUNCATED TO 500 CHARS):    at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasLookupInternal(String alias)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasInternal(String httpAlias)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequestForWebServer(String requestedPath)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequest(UrlAction result, String requestedPath, St</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="f21e4a04-b924-4c99-b36f-e1f049cd67c5" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 3:59:05 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of a change to the application level configuration.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="e735ab52-8d42-422d-8eb0-27dd45c851a8" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 3:59:07 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>e006ab1d-2fd9-498f-9d7b-3e69a1ebc7d5</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>zkytxFpN7gcAnkhdGESCMg==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Controllers.HostController.GetSettingsFromDatabase()</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="da14290b-ea28-4d8a-abde-2fd65e5d0f84" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 3:59:07 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>12f9bab3-3558-448f-8a33-9ea1af7d6b8d</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>KcxApFu8BKt81wwESI1ZDQ==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Portals.PortalAliasController.&lt;GetPortalAliasesInternal&gt;b__11(CacheItemArgs c)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromDictionary(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="cdc77d9d-7d16-4e39-928f-364c39fb4f4a" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 3:59:07 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>Url Processing Exception</PropertyName><PropertyValue>Exception in Url Rewriting Process</PropertyValue></LogProperty><LogProperty><PropertyName>Http Status</PropertyName><PropertyValue>500 Internal Server Error</PropertyValue></LogProperty><LogProperty><PropertyName>Original Path</PropertyName><PropertyValue>null</PropertyValue></LogProperty><LogProperty><PropertyName>Raw Url</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>Final Url</PropertyName><PropertyValue>null</PropertyValue></LogProperty><LogProperty><PropertyName>Rewrite Result</PropertyName><PropertyValue>[no rewrite]</PropertyValue></LogProperty><LogProperty><PropertyName>Redirect Location</PropertyName><PropertyValue>[no redirect]</PropertyValue></LogProperty><LogProperty><PropertyName>Action</PropertyName><PropertyValue>Continue</PropertyValue></LogProperty><LogProperty><PropertyName>Reason</PropertyName><PropertyValue>Exception</PropertyValue></LogProperty><LogProperty><PropertyName>Portal Id</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>Tab Id</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>Http Alias</PropertyName><PropertyValue>Null</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 1</PropertyName><PropertyValue>Exception: Value cannot be null.
Parameter name: source</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 2</PropertyName><PropertyValue>(TRUNCATED TO 500 CHARS): Stack Trace:    at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasLookupInternal(String alias)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasInternal(String httpAlias)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequestForWebServer(String requestedPath)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequest(UrlAction result, String requ</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 3</PropertyName><PropertyValue>Inner Ex : null</PropertyValue></LogProperty><LogProperty><PropertyName>Exception Type</PropertyName><PropertyValue>System.ArgumentNullException</PropertyValue></LogProperty><LogProperty><PropertyName>Message</PropertyName><PropertyValue>Value cannot be null.
Parameter name: source</PropertyValue></LogProperty><LogProperty><PropertyName>Stack Trace</PropertyName><PropertyValue>(TRUNCATED TO 500 CHARS):    at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasLookupInternal(String alias)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasInternal(String httpAlias)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequestForWebServer(String requestedPath)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequest(UrlAction result, String requestedPath, St</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="b02f0abd-2bce-490a-affe-8e95397bca57" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:00:29 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of a change to the application level configuration.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="d5f98b32-c744-47ad-b347-248317a8cdf3" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:00:31 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>cc10a8f5-f795-490b-866a-83e775057062</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>jYxmxaelNLo/OlQTJ9G+Og==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Controllers.HostController.GetSettingsFromDatabase()</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="ada65888-affb-42d1-abd6-b1a889c925c5" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:00:31 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>b0298c01-733c-44a3-a0d2-928d387508fb</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>R6ZTMNJBYm5CDFUzr18U7g==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Portals.PortalAliasController.&lt;GetPortalAliasesInternal&gt;b__11(CacheItemArgs c)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromDictionary(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="dabe5056-8a37-40a2-a51a-af895493620d" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:00:31 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>Url Processing Exception</PropertyName><PropertyValue>Exception in Url Rewriting Process</PropertyValue></LogProperty><LogProperty><PropertyName>Http Status</PropertyName><PropertyValue>500 Internal Server Error</PropertyValue></LogProperty><LogProperty><PropertyName>Original Path</PropertyName><PropertyValue>null</PropertyValue></LogProperty><LogProperty><PropertyName>Raw Url</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>Final Url</PropertyName><PropertyValue>null</PropertyValue></LogProperty><LogProperty><PropertyName>Rewrite Result</PropertyName><PropertyValue>[no rewrite]</PropertyValue></LogProperty><LogProperty><PropertyName>Redirect Location</PropertyName><PropertyValue>[no redirect]</PropertyValue></LogProperty><LogProperty><PropertyName>Action</PropertyName><PropertyValue>Continue</PropertyValue></LogProperty><LogProperty><PropertyName>Reason</PropertyName><PropertyValue>Exception</PropertyValue></LogProperty><LogProperty><PropertyName>Portal Id</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>Tab Id</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>Http Alias</PropertyName><PropertyValue>Null</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 1</PropertyName><PropertyValue>Exception: Value cannot be null.
Parameter name: source</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 2</PropertyName><PropertyValue>(TRUNCATED TO 500 CHARS): Stack Trace:    at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasLookupInternal(String alias)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasInternal(String httpAlias)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequestForWebServer(String requestedPath)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequest(UrlAction result, String requ</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 3</PropertyName><PropertyValue>Inner Ex : null</PropertyValue></LogProperty><LogProperty><PropertyName>Exception Type</PropertyName><PropertyValue>System.ArgumentNullException</PropertyValue></LogProperty><LogProperty><PropertyName>Message</PropertyName><PropertyValue>Value cannot be null.
Parameter name: source</PropertyValue></LogProperty><LogProperty><PropertyName>Stack Trace</PropertyName><PropertyValue>(TRUNCATED TO 500 CHARS):    at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasLookupInternal(String alias)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasInternal(String httpAlias)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequestForWebServer(String requestedPath)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequest(UrlAction result, String requestedPath, St</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="66919db4-6543-42e1-8fa9-f6c8373c2e6b" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:00:49 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of a change to the application level configuration.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="5b315c5b-0f23-4b73-80a9-e446f911086b" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:00:52 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>03f60930-335c-48d9-94e1-938a78451170</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>NTl2FxxAE2Q56X8AH8y1qw==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Controllers.HostController.GetSettingsFromDatabase()</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="0fe35089-5230-46b5-9d7f-c2e04cfe8d04" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:00:52 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>24261c62-9169-4c5f-9247-4f8834d3e671</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>pBLmfnvzFIOkJaOxC4HEog==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Portals.PortalAliasController.&lt;GetPortalAliasesInternal&gt;b__11(CacheItemArgs c)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromDictionary(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="4617430b-8654-47b6-a1cb-ec494fa31748" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:00:52 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>Url Processing Exception</PropertyName><PropertyValue>Exception in Url Rewriting Process</PropertyValue></LogProperty><LogProperty><PropertyName>Http Status</PropertyName><PropertyValue>500 Internal Server Error</PropertyValue></LogProperty><LogProperty><PropertyName>Original Path</PropertyName><PropertyValue>null</PropertyValue></LogProperty><LogProperty><PropertyName>Raw Url</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>Final Url</PropertyName><PropertyValue>null</PropertyValue></LogProperty><LogProperty><PropertyName>Rewrite Result</PropertyName><PropertyValue>[no rewrite]</PropertyValue></LogProperty><LogProperty><PropertyName>Redirect Location</PropertyName><PropertyValue>[no redirect]</PropertyValue></LogProperty><LogProperty><PropertyName>Action</PropertyName><PropertyValue>Continue</PropertyValue></LogProperty><LogProperty><PropertyName>Reason</PropertyName><PropertyValue>Exception</PropertyValue></LogProperty><LogProperty><PropertyName>Portal Id</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>Tab Id</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>Http Alias</PropertyName><PropertyValue>Null</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 1</PropertyName><PropertyValue>Exception: Value cannot be null.
Parameter name: source</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 2</PropertyName><PropertyValue>(TRUNCATED TO 500 CHARS): Stack Trace:    at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasLookupInternal(String alias)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasInternal(String httpAlias)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequestForWebServer(String requestedPath)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequest(UrlAction result, String requ</PropertyValue></LogProperty><LogProperty><PropertyName>Debug Message 3</PropertyName><PropertyValue>Inner Ex : null</PropertyValue></LogProperty><LogProperty><PropertyName>Exception Type</PropertyName><PropertyValue>System.ArgumentNullException</PropertyValue></LogProperty><LogProperty><PropertyName>Message</PropertyName><PropertyValue>Value cannot be null.
Parameter name: source</PropertyValue></LogProperty><LogProperty><PropertyName>Stack Trace</PropertyName><PropertyValue>(TRUNCATED TO 500 CHARS):    at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasLookupInternal(String alias)
   at DotNetNuke.Entities.Portals.PortalAliasController.GetPortalAliasInternal(String httpAlias)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequestForWebServer(String requestedPath)
   at DotNetNuke.Entities.Urls.AdvancedUrlRewriter.IgnoreRequest(UrlAction result, String requestedPath, St</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="bf4d492c-b938-4072-82d5-059b9b9f63e2" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:02:28 PM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>21ac19f0-39eb-4c7c-b480-6277c551fe9c</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>epuT7O1algEvcez2qRgMMQ==</ExceptionHash><Message>Login failed for user 'IIS APPPOOL\tomtang'.</Message><StackTrace>   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean&amp; dataReady)
   at System.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)
   at System.Data.SqlClient.SqlInternalConnectionTds.CompleteLogin(Boolean enlistOK)
   at System.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)
   at System.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)
   at System.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Portals.PortalAliasController.&lt;GetPortalAliasesInternal&gt;b__11(CacheItemArgs c)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromDictionary(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="0e0c2c77-9dbd-4471-a217-af3f95988f88" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="2/28/2015 4:11:06 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="Tom-MSI" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of a change to the application level configuration.</PropertyValue></LogProperty></LogProperties></log></logs>
